common/models/container_override.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+type ContainerOverride struct {
```

Comment:
this needs a swagger definition and to be defined in `controllers/swagger.go`.

I don't think there's a type for `maps` in swagger yet, so probably just say it's a `string` type for now and throw a `todo` statement.